### ISSUE TITLE
Report History

### ISSUE THREAD

**Issue by NilsG-S** [TTUSDC/cpceed-webapp#55]
We should consider having something to display report history. I.e. all the reports a given user has submitted. That way they can be sure a report hasn't gotten lost. It could also show the status. So they'd know if the report is accepted/pending/rejected.

@ynigoreyes

**Comment by ynigoreyes** [comment]
Wanna use blockchain to verify all reports lol?

**Comment by NilsG-S** [comment]
lol, that seems a bit excessive. We're not concerned about users falsely claiming to have submitted a report. We just don't want to leave them in suspense.

**Comment by ynigoreyes** [comment]
So what is a report is what exactly? Is it like a form that users can submit so that they can tell admins that they went somewhere?

**Comment by NilsG-S** [comment]
Yeah. It's basically a request for points on the basis of activity completed. There'll be some information inside the report to allow a coordinator to verify it.

**Comment by ynigoreyes** [comment]
okay okay, well it seems like activities and reports are very related. So in that case, we can make an activities/reports page and fill it up. We could do a small most recent activity feed about updated reports with a 'More' button that will open a list of their submitted reports in chronological order. The rest of the page can just be a big form for submitting activity reports.

**Comment by NilsG-S** [comment]
That sounds good to me
